plugins/item_tasks/plugin_tests/widgetsSpec.js

```diff
@@ -599,10 +601,52 @@ describe('control widget view', function () {
 
         // make sure the browser widget was initialized
         expect(browserWidgetProto.initialize).toHaveBeenCalled();
+        expect(browserWidgetProto.render).toHaveBeenCalled();
 
         expect(w.model.get('value')).toBe(item);
     });
 
+    it('file', function () {
```

Comment:
@brianhelba  I'm struggling to test this widget properly for a file. I believe it has something to do with the ItemSelectorWidget fetching the file before emitting a g:saved event.